examples/tutorials/self-paced-training/part-3_security_and_privacy/chapter-6_Security_in_federated_compute_system/06.5_communition_security/communication_security.ipynb

```diff
@@ -0,0 +1,195 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Connection and Message Security\n",
+    "\n",
+    "Currently Flare's message security comes from mutual TLS: server and client authenticate each other when making direct connections. This means that only clients that have the right startup kits can make a connection to the server.\n",
```

Comment:
Is this true? the main branch is 2.6, so it does not necessary mTLS?

Reply:
If we really want to say this here in the book, we should say:
"Before flare version 2.6, flare's message security comes from mutual TLS: xxx"

Reply:
I think this should be in the change log or PR description, but not necessary need to be in the tutorial.